engine/src/main/java/org/terasology/engine/logic/players/LocalPlayerSystem.java

```diff
@@ -263,6 +257,14 @@ public void updateRotationPitch(RotationPitchAxis event, EntityRef entity) {
         event.consume();
     }
 
+    @ReceiveEvent(components = CharacterComponent.class)
+    public void updateDirection(UpdateDirectionEvent event, EntityRef entity) {
+        if (localPlayer.getCharacterEntity().equals(entity)) {
+            lookPitch = event.getPitch();
+            lookYaw = event.getYaw();
+        }
```

Comment:
There are event handlers for `updateRotationYaw` and `updateRotationPitch`. If we were to use the same vocabulary, this would rather be `setRotation` (as `update...` is just setting the delta to be used in the next update).